libbeat/docs/command-reference.asciidoc

```diff
@@ -723,15 +732,27 @@ ifndef::no_dashboards[]
 {beatname_lc} setup --index-management
 -----
 endif::no_dashboards[]
+
+ifndef::apm-server[]
```

Comment:
Unfortunately I had to nest conditionals here. If you can think of a better idea, let me know.